### PR TITLE
Add fluid mappings and fix some inconsistencies and typo.

### PR DESCRIPTION
**Pull request by LambdAurora**
Renamed a lot of `BlockView` parameters named `world` to `view` to reduce inconsistency.
Fix a typo in `IronGolemEntity`.

### CODE REVIEW

**Comment by Pyrofab** [comment]
BlockView is most often implemented by `World`. I think the world parameter names should be kept.

**Comment by LambdAurora** [comment]
It's also implemented by `Chunk`.

It's currently very inconsistent, it would be good to find a standard.

File: mappings/net/minecraft/fluid/BaseFluid.mapping [hunk 1]
```diff
@@ -6,6 +6,9 @@ CLASS net/minecraft/class_3609 net/minecraft/fluid/BaseFluid
 		ARG 1 world
 		ARG 2 fluidPos
 		ARG 3 state
+	METHOD method_15726 (Lnet/minecraft/class_4538;Lnet/minecraft/class_2338;Lnet/minecraft/class_2680;)Ljava/util/Map;
```
Comment (liach) [inline mappings/net/minecraft/fluid/BaseFluid.mapping]: I suggest naming this method if possible

**Review by liach — APPROVED** [review]

File: mappings/net/minecraft/fluid/BaseFluid.mapping (hunk 1 above)
Comment (LambdAurora) [reply mappings/net/minecraft/fluid/BaseFluid.mapping]: I'm working on this, might take a little bit of time tho.

**Comment by liach** [comment]
Given result in #1098 let's use `world` than `view`

**Review by liach — APPROVED** [review]

**Comment by LambdAurora** [comment]
Now every `BlockView` argument is named `world` in fluid mappings at least.

And renamed the PR as it's a bit more than fixes now.

File: mappings/net/minecraft/fluid/LavaFluid.mapping [hunk 2]
```diff
@@ -1,3 +1,12 @@
 CLASS net/minecraft/class_3616 net/minecraft/fluid/LavaFluid
+	METHOD method_15817 hasBurnableBlock (Lnet/minecraft/class_4538;Lnet/minecraft/class_2338;)Z
+		ARG 1 world
+		ARG 2 pos
+	METHOD method_15818 playExtinguishEvent (Lnet/minecraft/class_1936;Lnet/minecraft/class_2338;)V
+		ARG 1 world
+		ARG 2 pos
+	METHOD method_15819 canPutFire (Lnet/minecraft/class_4538;Lnet/minecraft/class_2338;)Z
```
Comment (Runemoro) [inline mappings/net/minecraft/fluid/LavaFluid.mapping]: I think `canLightFire` sounds better.

Comment (LambdAurora) [reply mappings/net/minecraft/fluid/LavaFluid.mapping]: I agree with that, will push quickly!

**Review by Runemoro — APPROVED** [review]

**Comment by liach** [comment]
Isn't really a refactor (given this one changes param names which has no code-side effect)

File: mappings/net/minecraft/fluid/Fluid.mapping [hunk 3]
```diff
@@ -5,13 +5,26 @@ CLASS net/minecraft/class_3611 net/minecraft/fluid/Fluid
 	METHOD method_15774 getBucketItem ()Lnet/minecraft/class_1792;
 	METHOD method_15775 appendProperties (Lnet/minecraft/class_2689$class_2690;)V
 	METHOD method_15776 randomDisplayTick (Lnet/minecraft/class_1937;Lnet/minecraft/class_2338;Lnet/minecraft/class_3610;Ljava/util/Random;)V
+		ARG 1 world
+		ARG 2 pos
+		ARG 3 state
+		ARG 4 random
+	METHOD method_15777 canBeReplaceWith (Lnet/minecraft/class_3610;Lnet/minecraft/class_1922;Lnet/minecraft/class_2338;Lnet/minecraft/class_3611;Lnet/minecraft/class_2350;)Z
```
Comment (Virtuoel) [inline mappings/net/minecraft/fluid/Fluid.mapping]: `canReplaceWith` or `canBeReplacedWith` may be better here.

Comment (LambdAurora) [reply mappings/net/minecraft/fluid/Fluid.mapping]: Oops, made a typo here, I meant `canBeReplacedWith`.

**Review by Virtuoel — APPROVED** [review]

**Review by modmuss50 — APPROVED** [review]

**Pull request merged**